19.0/extra_requirements.txt

```diff
@@ -0,0 +1,48 @@
+# Extra python dependencies
```

Comment:
Into the images release-4.5.x, the `extra_requirements.txt` files have been removed and are now auto-installed.
https://github.com/camptocamp/docker-odoo-project/pull/355/files#diff-1b5f83ac426ad51f03a881ced21e9a83a923d3a7303963df88e0777e11e59cccR7